src/main/java/org/jenkinsci/plugins/parallel_test_executor/ParallelTestExecutor.java

```diff
@@ -122,7 +131,8 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         }
         FilePath dir = workspace.child("test-splits");
         dir.deleteRecursive();
-        List<InclusionExclusionPattern> splits = findTestSplits(parallelism, build, listener, includesPatternFile != null);
+        List<InclusionExclusionPattern> splits = findTestSplits(parallelism, build, listener, includesPatternFile != null,
```

Comment:
oh, and possibly time to introduce an overloaded method using a _Parameter Object_ pattern instead of adding yet another parameter.

Reply:
True, though this is package access so I think we do not much care.